guides/GETTING_STARTED.md

```diff
@@ -0,0 +1,69 @@
+
+For more information how to work with Crowdin please read official [guide for volunteer translators](https://support.crowdin.com/for-volunteer-translators/) or watch the [video](https://www.youtube.com/watch?v=bxdC7MfrO7A) prepared by Andriy Poznakhovskyy member of the Node.js i18n.
+
+### Step #1: Translation
+
+After successful log in to Crowdin and selecting project to translate find your locale from list and enjoy volunteering.
+
+> NOTE: If you cannot find your locale in list, please create a new issue in base repo ([see table](/#i18n-projects)) with mention to managers team
+
+All translations going to be approved by other translators, and then automatically pushed to GitHub repository, so you don't need to go outside Crowdin platform. Don't open pull requests to translate Node.js, and focus just on translation content.
+
+> NOTE: If you find an error in the original docs, feel free to open PR to the base repository ([see table](/#i18n-projects)).
+
+### Step #2: Proofread
+
+We really appreciate any contribution, but should validate translated content, so after you submitted your translation it's going to be approved by other translator with proofreader role. Find more details about roles in [roles guideline](./ROLES.md).
+
+### Step #3: Sync
+
+Crowdin automatically create PR with updates to base GitHub repository around 24 hours after updates was approved. If you think that's something goes wrong with sync process, please create a new issue with managers mention.
+
+## Communication process
+
+We propose a few ways to communicate:
+
+- Slack:
+  - [node-js/#i18n](https://node-js.slack.com/archives/C8S7FCNR1) - for general questions
+  - [openjs-foundation/#i18n](https://openjs-foundation.slack.com/archives/CUH8WBHBL) - for general questions
+- Crowdin:
+  - [conversations](https://support.crowdin.com/conversations/) - direct and group chats to communicate with other translators
+  - [comments](https://support.crowdin.com/online-editor/#communicating-with-other-project-members) - discuss translation directly
+- GitHub:
+  - [nodejs/i18n](https://github.com/nodejs/i18n) - create issue/PR if you have not language specific question
+  - l10n repositories and teams - for language/team specific questions, find full list in [teams guideline](https://github.com/nodejs/i18n/blob/master/TEAMS.md)
+
+Feel free to choose most comfortable way for your goals, but we'd like to recommend to use GitHub for most cases. It's standardized and controlled by Node.js community.
+
+Localization process for each language have specific questions, that should be discussed and agreed somewhere, and for these reasons we've created l10n teams for each current active locale. Main aim is to gather people interested in taking part in the translation process in one place, so that it can be used as a source of help when problems/doubts arise during the translation process. To contact a specific l10n team, all you need is an account on GitHub.
```

Comment:
```suggestion
The localization process for each language can have specific questions that should be discussed somewhere. For this reason we've created l10n teams for each current active locale. The main goal is to gather people interested in taking part in the translation process in one place so that it can be used as a source of help when problems/doubts arise during the translation process. To contact a specific l10n team, all you need is an account on GitHub.
```